flaskr/routes.py

```diff
@@ -88,15 +94,17 @@ def listprescriptions():
     prescriptions = Prescription.query.all()
     prescription_schema = PrescriptionSchema(many=True)
     ser_prescriptions = prescription_schema.dump(prescriptions)
-    for prescription in ser_prescriptions:
-        medications = prescription['medications']
-        for medication in medications:
-            med_data = getMedicationData(medication['product_ndc'])
+    # this wont work without the product_nd - will revise what to send back
```

Comment:
Was the goal here to query the API for all the particulars for a drug? Every manufacturer has one or more NDCs for a particular manufacture. How would someone know what is what? I think this reflects too much granularity.

Reply:
exactly, hence the reason I removed it for now. We dont need to include it.